sdk/quantum/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Temp files used to auto-generate code
```

Comment:
This file can be removed. Use the `.gitignore` file in the root of the repo if required.